python/servo/testing_commands.py

```diff
@@ -245,24 +245,29 @@ def test_unit(self, test_name=None, package=None, bench=False, nocapture=False,
 
         packages.discard('stylo')
 
-        if len(packages) > 0 or len(in_crate_packages) > 0:
-            args = []
-            for crate in packages:
-                args += ["-p", "%s_tests" % crate]
-            for crate in in_crate_packages:
-                args += ["-p", crate]
-            args += test_patterns
-
-            if nocapture:
-                args += ["--", "--nocapture"]
-
-            err = self.run_cargo_build_like_command("bench" if bench else "test",
-                                                    args,
-                                                    env=self.build_env(test_unit=True),
-                                                    with_layout_2020=with_layout_2020,
-                                                    **kwargs)
-            if err:
-                return err
+        # Return if there is nothing to do.
+        if len(packages) == 0 and len(in_crate_packages) == 0:
+            return 0
+
+        # https://internals.rust-lang.org/t/exploring-crate-graph-build-times-with-cargo-build-ztimings/10975
+        # Prepend so that e.g. `-Ztimings` (which means `-Ztimings=info,html`)
```

Comment:
did you mean to say `-Ztimings=info.html` ?

Reply:
Oh, I copied the comment from the same lines that are used to set this up for `./mach build`, but the comment is seriously out of date. I'll update it and the old one as well.